test/unit/gateways/pay_trace_test.rb

```diff
@@ -29,6 +29,51 @@ def test_successful_purchase
     assert_equal 392483066, response.authorization
   end
 
```

Comment:
I'm wondering if there should be a unit test showing successful mapping of level_3 data, including things like `source_address`, `shipping_address`, and/or multiple `line_items`?

Reply:
I've got an additional remote test built now with multiple line items that also tests the mapping of `address` to `source_address`, as well as a new unit test 🎉